### PR TITLE
Use the most recent redeyed 1.0

### PR DESCRIPTION
**Pull request by ariya**

### CODE REVIEW

File: test/cardinal-highlight-file-sync.js [hunk 1]
```diff
@@ -29,13 +29,10 @@ test('not supplying custom theme', function (t) {
...
-test('errornous code', function (t) {
-  try {
-    cardinal.highlightFileSync(fileWithErrors);
-  } catch (e) {
-    t.similar(e.message, /Unable to perform highlight. The code contained syntax errors.* Line 1: Unexpected token [(]/)
-    t.end()
-  }
+test('syntactically invalid code', function (t) {
+  var highlighted = cardinal.highlightFileSync(fileWithErrors);
+  t.equals(highlighted, '\u001b[94mfunction\u001b[39m \u001b[90m(\u001b[39m\u001b[90m)\u001b[39m \u001b[33m{\u001b[39m \n  \u001b[32mvar\u001b[39m \u001b[37ma\u001b[39m \u001b[93m=\u001b[39m \u001b[34m3\u001b[39m\u001b[90m;\u001b[39m \u001b[31mreturn\u001b[39m \u001b[37ma\u001b[39m \u001b[93m>\u001b[39m \u001b[34m2\u001b[39m \u001b[93m?\u001b[39m \u001b[91mtrue\u001b[39m \u001b[93m:\u001b[39m \u001b[91mfalse\u001b[39m\u001b[90m;\u001b[39m \n\u001b[33m}\u001b[39m\u001b[90m;\u001b[39m\n')
```
Comment (ariya) [inline test/cardinal-highlight-file-sync.js]: Non-parseable code isn't a problem anymore, thanks to redeyed 1.0!

File: test/cardinal-highlight-json.js [hunk 2]
```diff
@@ -32,10 +32,8 @@ test('not supplying custom theme', function (t) {
...
 test('without json option', function (t) {
-  try {
-    cardinal.highlight(json);
-  } catch (e) {
-    t.similar(e.message, /Unable to perform highlight. The code contained syntax errors.* Line 1: Unexpected token /)
-    t.end();
-  }
+  var highlighted = cardinal.highlight(json);
+
+  t.equals(highlighted, '\u001b[33m{\u001b[39m\u001b[32m"foo"\u001b[39m\u001b[93m:\u001b[39m\u001b[92m"bar"\u001b[39m\u001b[32m,\u001b[39m\u001b[32m"baz"\u001b[39m\u001b[93m:\u001b[39m\u001b[92m"quux"\u001b[39m\u001b[32m,\u001b[39m\u001b[32m"bam"\u001b[39m\u001b[93m:\u001b[39m\u001b[90mnull\u001b[39m\u001b[33m}\u001b[39m')
```
Comment (ariya) [inline test/cardinal-highlight-json.js]: With redeyed 1.0, the `json` flag in the option is unnecessary anymore.

@thlorenz Do you like me to remove it from the documentation as well?

Comment (thlorenz) [inline test/cardinal-highlight-json.js]: Why isn't it necessary? Is esprima detecting things for us?
And would we treat `.json` files the same way as `.js` files? (see [here](https://github.com/thlorenz/cardinal/blob/master/lib/highlightFile.js#L14) and [here](https://github.com/thlorenz/cardinal/blob/master/lib/highlightFileSync.js#L7)

If so let's deprecate that option and remove it from these places as well as the docs.

Comment (ariya) [inline test/cardinal-highlight-json.js]: With the pure tokenizer, JSON and JS are not treated differently (a tokenizer can't know the semantics).

I'll remove the JSON logic detection and update the docs.

Comment (thlorenz) [inline test/cardinal-highlight-json.js]: Makes sense, thanks.

**Comment by thlorenz** [comment]
LGTM, please merge to master.

**Comment by ariya** [comment]
Thanks @thlorenz! I don't think I have merge access, though.

**Comment by thlorenz** [comment]
Sorry, @ariya fixed that .. now you should be able to merge :)

**Comment by thlorenz** [comment]
Merging this as I just used this version locally on minified JSON with intermixed text output and was impressed!
Tools like `jq` and such fail miserably in those cases, but even if the JSON stays minified the colorization makes it readable.
However possibly we can add some option to format it nicely? Or maybe even do so by default?
What do you think?

**Comment by thlorenz** [comment]
Merged and published as 1.0, thanks a lot!

**Comment by ariya** [comment]
Thanks for merging @thlorenz! I'm not sure I'm following you regarding the JSON formatting. Perhaps open an issue so we can discuss it in details?

**Pull request closed without merging**